bin/travis/before_install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+sudo install -o "$USER" -d /etc/blocktogether
+sudo install -o "$USER" -d /data/blocktogether
+cp -a "$(dirname ${BASH_SOURCE[0]})/../../"config/* /etc/blocktogether/
+sed "s/XXXXXXXXXXXXXXXXXXXXXXXXX/$CONSUMER_KEY/" \
+    "$(dirname ${BASH_SOURCE[0]})/../../"config/production.json > /etc/blocktogether/config.json
+sed -i "s/YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY/$CONSUMER_SECRET/" /etc/blocktogether/config.json
+# turn off secureProxy
+sed -i 's/true/false/' /etc/blocktogether/config.json
+# use local callback
+sed -i 's|https://blocktogether.org/auth/twitter/callback|http://localhost:2078/auth/twitter/callback|' /etc/blocktogether/config.json
```

Comment:
I did use the production config because I forgot that Vagrant also uses `setup.sh`, and wasn't sure if it was configured to work with the dev config.